third_party/move/move-model/src/metadata.rs

```diff
@@ -93,12 +89,7 @@ pub enum CompilerVersion {
 impl Default for CompilerVersion {
     /// We allow the default to be set via an environment variable.
     fn default() -> Self {
-        static MOVE_COMPILER_V2: Lazy<bool> = Lazy::new(get_move_compiler_v2_from_env);
-        if *MOVE_COMPILER_V2 {
-            Self::V2_0
-        } else {
-            Self::V1
-        }
+        Self::latest_stable()
```

Comment:
I wonder if it is still useful to allow setting the compiler version to 1 via env variable (to handle those scenarios where `--move-1` is not applicable).